src/tests/ftest/util/ior_utils.py

```diff
@@ -159,9 +162,7 @@ def set_daos_params(self, group, pool, cont_uuid=None, display=True,
 
         # assigning obj class as SX in None else
         # the desired one
-        if mpiio_oclass is None:
-            self.mpiio_oclass = 214
-        else:
+        if mpiio_oclass is not None:
```

Comment:
ditto.  Plus I'm not sure this would yield the user expected experience as passing a different `mpiio_oclass` value into this function would not only end up calling ior with a value different than `self.mpiio_oclass` (expected) but it will also reassign `self.mpiio_oclass` (probably unexpected) - but this would be moot if using self.daos_oclass to set the env.